pkg/openstack/common.go

```diff
@@ -47,6 +47,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	// ooSelector is used as aselector to the labels to differentiate from
+	// any other possible labels added to services, because  e.g. the common.AppSelector
+	// is also used by service operators.
+	ooSelector = "osctlplane"
```

Comment:
where is this used?
I thought it would have been a value passed to ` GetServiceOpenStackOperatorLabel` to extend the selector, but I see an empty value assigned to the key

Reply:
it is applied to all services via the overrides so that the openstack-operator has another selector to know which services created by service operators are the ones which are used to expose the service and needs to take care of. e.g. the swift, but also the glance operator creates additional services which should be ignored when looking for services